tests/python/unittest/test_operator.py

```diff
@@ -7009,6 +7009,13 @@ def test_invalid_max_pooling_pad_type_same():
         name='pooling',
         pooling_convention="same")
 
+@with_seed()
+def test_randint():
```

Comment:
why not include this test along with the other random samplers in the `test_random.py` file here - https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_random.py#L32

Reply:
ya i was contemplating about it. ended up misplacing it coz thought this is a separate operator so should be put in test_operator instead. Will put it in test_random